spec/features/users_spec.rb

```diff
@@ -25,7 +25,7 @@
       visit "/users/#{user.username}"
     end
 
-    it 'チェックイン情報が表示されること', js: true do
+    it '記録情報が表示されること', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.